apps/prairielearn/src/server.ts

```diff
@@ -1453,34 +1453,20 @@ export async function initExpress(): Promise<Express> {
     (await import('./pages/studentAssessmentInstance/studentAssessmentInstance.js')).default,
   );
 
+  // Perform auth for all student-facing instance question routes.
```

Comment:
In #10906, we pushed these middlewares down into individual pages. However, because of the way that Express decides to execute middleware, these were _also_ running for all pages with this `.../instance_question/:instance_question_id/` path prefix. I ended up moving them back here to hopefully make it clearer that they're intentionally running for all of these paths, as it was really confusing to me that middleware mounted to this sub-router was running for pages on unrelated routers.